app/models/concerns/offeror.rb

```diff
@@ -0,0 +1,99 @@
+module Offeror
+
+	def self.import(file, current, comment)
```

Comment:
some very long methods - could we split out sub-methods and give them self-documenting names?